src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/WorkflowRunController.kt

```diff
@@ -124,12 +130,13 @@ class WorkflowRunController(
                             @Suppress("UnsafeCallOnNullableType")
                             context.userProfile!!.id,
                             Instant.now(),
-                            workflowRunRequest.reports.zip(workflowRun.reports) { report, reportKey ->
+                            workflowRun.reports.map {
```

Comment:
we now get back a more complete report object from orderly server, so no need to cross reference these with the original request at all